Objects/typeobject.c

```diff
@@ -4707,6 +4707,9 @@ static PyMethodDef object_methods[] = {
     {0}
 };
 
+PyDoc_STRVAR(object_doc,
+"object()\n--\n\nThe starting base class of all types and classes other\n"
```

Comment:
I am not familiar with docstrings in C code, but it appears that 'classname()\n--\n\n' gets removed before making the rest into the class object __doc__ attribute.